test/controllers/microposts_controller_test.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class MicropostsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @micropost = microposts(:one)
+  end
+
+  test 'should get index' do
```

Comment:
use our test naming convention. Llink to our dev practices is on cheat sheet. For this it would be 'should succeed get index'

Reply:
Also lots of examples in IO and DM controller tests

Reply:
Apply to all your controller tests.